.gitignore

```diff
@@ -38,3 +38,10 @@ de.metas.report/report-service/application.properties
 #Ignore IDEA files
 *.iml
 **/.idea
+de.metas.material/dispo-service/application.properties
+vertical-healthcare_ch/forum_datenaustausch_ch.invoice_base/LICENSE.txt
```

Comment:
if you want, add `LICENSE.txt` to gitignore by using a wildcard, but i think adding these few explicit files doesn't make a lot of sense